contracts/lib/CampaignLibrary.sol

```diff
@@ -3,8 +3,11 @@ pragma solidity ^0.4.19;
 
 library CampaignLibrary {
 
+    event CountryIndexes(uint[3] countries);
```

Comment:
Is this event is called somewhere? It does not serve a propose since we are already showing the campaign country codes on the create or update campaign.

Reply:
My bad, I created it for debug. There is no need for this event now.